api/groups/eventsGroup_test.go

```diff
@@ -195,11 +174,9 @@ func TestEventsGroup_PushEvents(t *testing.T) {
 
 		wasCalled := false
 		args := createMockEventsGroupArgs()
-		args.Facade = &mocks.FacadeStub{
-			HandlePushEventsV1Called: func(eventsData data.SaveBlockData) error {
-				return common.ErrReceivedEmptyEvents
-			},
-			HandlePushEventsV2Called: func(events data.ArgsSaveBlockData) error {
+		args.PayloadHandler = &testscommon.PayloadHandlerStub{
+			ProcessPayloadCalled: func(payload []byte, topic string) error {
+				require.Equal(t, topic, outport.TopicSaveBlock)
```

Comment:
[optional]
The tests would've been more comprehensive if we were to also check the payload
!Applies globally in this file, if of interest

Reply:
added additional checks